spec/controllers/spree/user_sessions_controller_spec.rb

```diff
@@ -30,4 +30,13 @@
       end
     end
   end
+
+  describe "destroy" do
+    it "redirects to root with flash message" do
+      spree_post :destroy
+
+      expect(response).to redirect_to root_path
+      expect(flash[:notice]).to eq "Signed out successfully."
```

Comment:
Hey @SarvarKh ,
I'm wondering, which flash notice is this line testing exactly?

I was expecting to find something like `[:flash] = t('devise.sessions.signed_out')` somewhere in the code or in this PR...

The reason I'm asking is that I could not get the translation to work on this PR, neither on a staging server nor locally, after adding the respective keys for the different locales.

Maybe everything is fine with the PR, just I could not observe that it is closing the issue. Thanks for your help on this.

Reply:
@filipefurtad0 The code which sets those messages is tucked away in the Devise gem, it doesn't live in our codebase. Did you restart puma after changing the translations locally?

Reply:
Ah I see, thanks Matt.

I did restart puma with `bundle exec pumactl -F /home/openfoodnetwork/apps/openfoodnetwork/shared/config/puma.rb start` on the staging server, and locally also stopped and restarted the rails server, after adding those keys to `en_CA.yml` and `fr_CA.yml` on staging UK.

Observed the change while using the app in English, but selecting French basically displayed `en_CA.yml`, which is the default language on the server. So yeah, I was wondering if something was missing.

But yeah, nothing breaks so I guess we can merge and check again if it works in production?

Reply:
I did a quick dev-test and changed those keys in `en.yml` and saw the changes as expected, so it's definitely using them :shrug: 

I think we can merge it :+1:

Reply:
Cool, thanks for double checking!

Reply:
Thanks @filipefurtad0 and @Matt-Yorkley 🙏